packages/cli/src/commands/composite/from-model.ts

```diff
@@ -0,0 +1,58 @@
+import { Composite } from '@glazed/devtools'
+import { Command, type CommandFlags } from '../../command.js'
+import { Flags } from '@oclif/core'
+import { writeEncodedComposite } from '@glazed/devtools-node'
+
+type Flags = CommandFlags & {
+  output?: string
+}
+
+export default class CompositeFromModel extends Command<Flags> {
+  static strict = false
+
+  static description = 'create a Composite from a list of model streams'
+
+  static flags = {
+    ...Command.flags,
+    output: Flags.string({
+      char: 'o',
+      description: 'path to the file where the composite representation should be saved',
+    }),
+  }
+
+  async run(): Promise<void> {
+    const parsed = await this.parse(CompositeFromModel)
+    const modelStreamIDs = parsed.raw
+      .filter((token) => {
+        return token.type === 'arg'
+      })
+      .map((token) => {
+        return token.input
+      })
+
+    if (modelStreamIDs.length === 0) {
+      this.spinner.fail('Missing list of model streamIDs')
+      return
+    }
+
+    try {
+      const composite = await Composite.fromModels({
+        ceramic: this.ceramic,
+        models: modelStreamIDs,
+      })
+      if (this.flags.output !== undefined) {
```

Comment:
```suggestion
      if (this.flags.output != null) {
```
Please make the checks less strict so possible `null` values don't go in this branch.